packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1113,32 +1153,30 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     /**
-     * Notifies this object about the request made to the container.
-     * @param request - Request made to the handler.
+     * @deprecated - use getEntryPointInstead
      */
-    public async request(request: IRequest): Promise<IResponse> {
-        try {
-            const parser = RequestParser.create(request);
-            const id = parser.pathParts[0];
-
-            if (id === "_summarizer" && parser.pathParts.length === 1) {
-                if (this._summarizer !== undefined) {
-                    return {
-                        status: 200,
-                        mimeType: "fluid/object",
-                        value: this.summarizer,
-                    };
-                }
-                return create404Response(request);
-            }
-            if (this.requestHandler !== undefined) {
-                return this.requestHandler(parser, this);
+    async request(request: IRequest): Promise<IResponse> {
+        const parser = RequestParser.create(request);
+        const id = parser.pathParts[0];
+
+        // todo - this should move somewhere else so we
+        // don't need to request to get it
```

Comment:
current idea is to make it the default entry point of runtimes that are summarizers